service_mesh/service_mesh_install/installing-mt-ossm.adoc

```diff
@@ -0,0 +1,24 @@
+[id="installing-mt-ossm"]
+= Installing {ProductName} with multi-tenant control planes
+include::modules/ossm-document-attributes.adoc[]
+:context: installing-mt-ossm
+toc::[]
+
+The {ProductName} operator supports multi-tenant control plane installations.
+
+[NOTE]
+====
+You cannot use multi-tenant control plane installations in conjunction with a cluster-wide control plane installation. {ProductName} installations must either be multi-tenant or single, cluster-wide installations.
+====
+
```

Comment:
If you have to follow the other installing assembly first, I'd add an assembly-level prereqs section and xref to that assembly.

Reply:
Done and done.

Note: in the near future multi-tenancy will be the default state for OSSM so this will all change again.